### PR TITLE
Search weight boost

### PR DESCRIPTION
**Pull request by wanliAlex**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Add the search weight boost support at py-marqo 

* **What is the current behavior?** (You can also link to an open issue here)
not search weight boost

* **What is the new behavior (if this is a feature change)?**
add search weight boost

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:
this PR is related to https://github.com/marqo-ai/marqo/pull/300

### CODE REVIEW

File: tests/v0_tests/test_boost_search.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                                                                     boost={"Title": [1, 1]})
+        res = self.client.index(self.index_name_1).search(q=self.query)
+
+        boost_score = boost_res["hits"][0]["_score"]
+        res_score_positive = res["hits"][0]["_score"]
+
+        self.assertEqual(boost_score, res_score_positive * 1 + 1)
+
+
+    def test_boost_search_query_forat(self):
```
Comment (pandu-k) [inline tests/v0_tests/test_boost_search.py]: typo. Did you mean `...query_format`?

**Comment by wanliAlex** [comment]
image

**Review by pandu-k — APPROVED** [review]
- One comment!

File: tests/v0_tests/test_boost_search.py (hunk 1 above)
Comment (wanliAlex) [reply tests/v0_tests/test_boost_search.py]: updated

**Comment by wanliAlex** [comment]
search weight boost

**Pull request merged**